app/models/enumeration/SchemeAdministratorType.scala

```diff
@@ -16,13 +16,14 @@
 
 package models.enumeration
 
-import play.api.mvc.PathBindable
+import models.enumeration
 import models.enumeration.binders.EnumPathBinder
+import play.api.mvc.PathBindable
 
 object SchemeAdministratorType extends Enumeration {
   type SchemeAdministratorType = Value
-  val PSA = Value("PSA")
-  val PSP = Value("PSP")
+  val PSA: enumeration.SchemeAdministratorType.Value = Value("PSA")
```

Comment:
Have you been able to test that this still works?

Reply:
Tested by starting a new AFT return and seems to work fine